build.gradle

```diff
@@ -62,6 +62,7 @@ ext {
   // Libraries
   nettyVersion = '4.1.34.Final'
   jacksonDatabindVersion = '2.5.1'
+  braveVersion = "5.6.1"
```

Comment:
switch to 5.8.0 for some more powerful things.. especially allows secondary sampling